doc/visual-programming/source/widgets/data/file.rst

```diff
@@ -55,8 +54,8 @@ both the :doc:`Data Table<../data/datatable>` and the :doc:`Box Plot<../visualiz
 Loading your data
 -----------------
 
--  Orange can import any comma, .xlsx or tab-delimited data file. Use
-   :doc:`File widget<../data/file>` and then, if needed, select class and meta attributes in
+-  Orange can import any comma, .xlsx or tab-delimited data file or URL. Use the
+   :doc:`File widget<../data/file>` and then, if needed, select class and meta attributes in the
```

Comment:
They can do that now already in the File widget itself. I'd fix that and add an example.